### PR TITLE
git-extras: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

**Comment by CleanMachine1** [comment]
This is going to make searching for #5137 annoying!

**Comment by bl-ue** [comment]
Why?

File: pages/common/git-extras.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Set of commands for `Git`.
```
Comment (CleanMachine1) [inline pages/common/git-extras.md:3]:
```suggestion
> Set of commands for `git`.
```

Comment (bl-ue) [inline pages/common/git-extras.md:3]:
```suggestion
> Set of commands for Git.
```

File: pages/common/git-extras.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://github.com/tj/git-extras>.
+
+- Download and or upgrade `git-extras` commands:
```
Comment (CleanMachine1) [inline pages/common/git-extras.md:6]:
```suggestion
- Download and or upgrade `git-extras` commands:
```

This is correct Engrish to my knowledge however what do you think

Comment (bl-ue) [inline pages/common/git-extras.md:6]: This is a little bit confusing although I know what you mean.  This is less so but still covers the same breadth.
```suggestion
- Update `git-extras`:
```

File: pages/common/git-extras.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`git extras update`
+
+- Display help and commands for `git-extras`:
```
Comment (bl-ue) [inline pages/common/git-extras.md:10]:
```suggestion
- Display help and subcommands:
```

File: pages/common/git-extras.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/git-extras.md:3]: Git is really almost like a brand, not a technical term.

File: pages/common/git-extras.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/git-extras.md:6]: With the ubuntu repo, which is what most are probably using, the `git-extras` package doesn't come with any commands to my knowledge, so its probably worth leaving that it downs the commands

File: pages/common/git-extras.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/git-extras.md:3]: See [my suggestion](https://github.com/tldr-pages/tldr/pull/6037#discussion_r637580012) below, which is more appropriate.

File: pages/common/git-extras.md (hunk 2 above)
Comment (bl-ue) [reply pages/common/git-extras.md:6]: That suggestion doesn't change anything, @CleanMachine1.

Comment (bl-ue) [reply pages/common/git-extras.md:6]: Also, I've never (to my knowledge) heard or spoke Engrish, can you enlighten me?

Comment (CleanMachine1) [reply pages/common/git-extras.md:6]: It was ironic.. 
i didn't mean to use the suggestion, muscle memory.

is the `and or` acceptable

Comment (bl-ue) [reply pages/common/git-extras.md:6]: "and" followed by "or" is perfectly fine, but I'd prefer a solidus in between:
```suggestion
- Download and/or upgrade `git-extras` commands:
```

Comment (bl-ue) [reply pages/common/git-extras.md:6]: Good point — let's skip this and apply [this](https://github.com/tldr-pages/tldr/pull/6037#discussion_r637580876) instead.

Comment (CleanMachine1) [reply pages/common/git-extras.md:6]: 14 files have it, committed.

File: pages/common/git-extras.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Set of commands for Git.
```
Comment (marchersimon) [inline pages/common/git-extras.md:3]: `Unofficial Git extension pack.` would also be nice. 😄

File: pages/common/git-extras.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Download and/or upgrade `git-extras` commands:
+
+`git extras update`
+
+- Display help and subcommands:
+
+`git extras --help`
+
+- Print version of installed `git-extras`:
```
Comment (marchersimon) [inline pages/common/git-extras.md:14]:
```suggestion
- Print the version of installed `git-extras`:
```

File: pages/common/git-extras.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/git-extras.md:3]: Yeah, I the current description is way too bland.  "Unofficial" though is too negative, I think 😝

How about that minus "unofficial"?
```suggestion
> Git extension pack.
```

**Review by marchersimon — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Could we possibly rename this PR to git-3xtras: add page

**Comment by bl-ue** [comment]
I don't think that's necessary.  A search for [`git extras`](https://github.com/tldr-pages/tldr/issues?q=git+extras) reveals only 14 items, two of which are documented with `help wanted`.

**Comment by CleanMachine1** [comment]
Ok, then I'll live with it :)

**Comment by bl-ue** [comment]
It _is_ a bit annoying, but I really can't think of any way around it.

**Comment by CleanMachine1** [comment]
_ ? 

I often do "#git-extras" {enter}

**Comment by bl-ue** [comment]
Hmm, I'm confused.  I though you were speaking of the fact that a serch for `git extras` returns #5137 and #6037 (a 900 difference! :), which is confusing.  Am I incorrect?

**Comment by CleanMachine1** [comment]
When I am changing/editing #5137 I need to link the PRs and now there is the chance of error when pressing enter on the first result

**Comment by bl-ue** [comment]
Ah, I see!  That doesn't seem like much of a concern, @CleanMachine1.  If you make a mistake and send the comment with the wrong link, who cares! it's not the end of the world :)

I didn't know that you can type text after the `#` 🤔 TIL!

**Comment by CleanMachine1** [comment]
Really! I thought everyone would know that one, thats probably one of my most used features. 
Do you remember PR #'s

**Comment by bl-ue** [comment]
Yeah, I memorize the most common ones.  Otherwise I open the PR/issue in a new tab, copy its number, switch back, paste 😆

**Comment by CleanMachine1** [comment]
Welp, thats going to save you some mouse clicks!
YW

**Comment by bl-ue** [comment]
It makes sense for it to exist, I'm just dumb 🤷🏻‍♂️

**Comment by CleanMachine1** [comment]
Are you going to approve this PR

**Review by bl-ue — APPROVED** [review]

File: pages/common/git-extras.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Git extension pack.
+> More information: <https://github.com/tj/git-extras>.
+
+- Download and/or upgrade `git-extras` commands:
```
Comment (navarroaxel) [inline pages/common/git-extras.md:6]: I read the thread about this example description, but why just `download`? why not?
```suggestion
- Install or upgrade `git-extras` commands:
```
`Download` sounds like `get a tarball file in the current directory` :man_shrugging:

Comment (CleanMachine1) [reply pages/common/git-extras.md:6]: Sure.

Comment (navarroaxel) [reply pages/common/git-extras.md]: Sounds ok to you @bl-ue?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**